src/zenml/cli/model.py

```diff
@@ -619,25 +584,25 @@ def list_model_version_data_artifacts(
     "model_artifacts",
     help="List model artifacts linked to a model version.",
 )
-@click.argument("model_name_or_id")
-@click.argument("model_version_name_or_number_or_id", default="0")
```

Comment:
Don't like this change in the way it is done now. Can we move this up to model group? Version group assume that version is an argument in all methods.